smelter-core/src/pipeline/rtp/rtp_input.rs

```diff
@@ -270,9 +273,33 @@ fn run_rtp_demuxer_thread(
         }
     };
     loop {
-        let Ok(mut buffer) = receiver.recv() else {
-            debug!("Closing RTP demuxer thread.");
-            break;
+        let mut buffer = match receiver.recv_timeout(Duration::from_millis(10)) {
+            Ok(buffer) => buffer,
+            Err(RecvTimeoutError::Timeout) => {
+                if let Some(video) = &mut video {
```

Comment:
Normally, this would not need to be triggered, but our tests are sending a lot of data at the start, so recv is never called latter and data is not pop-ed from jitter buffer